test/e2e/tests/test_api_keys.py

```
@@ -1346,8 +1360,18 @@ def test_reject_key_for_unreconciled_subscription(self):
             log.info("✅ API key creation rejected for unreconciled subscription")
 
         finally:
+            # Restore webhook failurePolicy to Fail
+            subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)